crates/biome_css_formatter/tests/specs/prettier/css/numbers/numbers.css.snap

```diff
@@ -94,122 +93,67 @@ info: css/numbers/numbers.css
    a {
      a: 0;
      a: 1;
-@@ -6,70 +6,72 @@
-     a: 0.1;
+@@ -7,14 +7,14 @@
      a: 1.1;
  
--    a: 0.1;
+     a: 0.1;
 -    a: 1;
--    a: +0.1;
--    a: -0.1;
-+    a: .1;
 +    a: 1.;
-+    a: +.1;
-+    a: -.1;
+     a: +0.1;
+     a: -0.1;
  
      a: 1e1;
--    a: 1e1;
-+    a: 1e+1;
+     a: 1e1;
      a: 1e-1;
 -    a: 1e1;
--    a: 0.1e1;
 +    a: 1.e1;
-+    a: .1e1;
+     a: 0.1e1;
      a: 1.1e1;
--    a: 1.1e10;
--    a: +1.1e10;
--    a: -1.1e10;
--    a: 0.1e10;
--    a: 0.1e-10;
-+    a: 1.1e0010;
-+    a: +1.1e0010;
-+    a: -1.1e0010;
-+    a: .1e+0010;
-+    a: .1e-0010;
- 
--    a: 1e1;
--    a: 1e1;
--    a: 1e-1;
+     a: 1.1e10;
+@@ -26,7 +26,7 @@
+     a: 1e1;
+     a: 1e1;
+     a: 1e-1;
 -    a: 1e1;
--    a: 0.1e1;
--    a: 1.1e1;
--    a: 1.1e10;
--    a: 0.1e10;
--    a: 0.1e-10;
-+    a: 1E1;
-+    a: 1E+1;
-+    a: 1E-1;
 +    a: 1.E1;
-+    a: .1E1;
-+    a: 1.1E1;
-+    a: 1.1E0010;
-+    a: .1E+0010;
-+    a: .1E-0010;
```

Comment:
I'm not sure this is correct. It should print with lowercase `e`

Reply:
Yes, you're right, it's a problem how github shows changes =(

However there is a problem with a number parsing and I'm unsure how to handle it better.
```
a: 1.;
a: 0.e60;
```
our parser can't parse these number because according to spec: https://drafts.csswg.org/css-syntax-3/#consume-number

If the [next 2 input code points](https://drafts.csswg.org/css-syntax-3/#next-input-code-point) are U+002E FULL STOP (.) followed by a [digit](https://drafts.csswg.org/css-syntax-3/#digit), then:
Consume the [next input code point](https://drafts.csswg.org/css-syntax-3/#next-input-code-point) and append it to number part.
While the [next input code point](https://drafts.csswg.org/css-syntax-3/#next-input-code-point) is a [digit](https://drafts.csswg.org/css-syntax-3/#digit), consume it and append it to number part.
Set type to "number".